mlir/lib/Tools/PDLL/Parser/Parser.cpp

```diff
@@ -314,9 +314,11 @@ class Parser {
   FailureOr<ast::Expr *> parseExpr();
 
   /// Identifier expressions.
+  FailureOr<ast::Expr *> parseArrayAttrExpr();
```

Comment:
I think it makes sense to rename this to `parseArrayExpr`, such that it mirrors how the other parsing functions are called

Reply:
What about keep the `parseArrayAttrExpr` and rename the other one to `parseDictAttrExpr`?